### PR TITLE
Add faq to detail page

### PR DESCRIPTION
**Pull request by Simsteve7**
#317

### CODE REVIEW

File: tweb/src/main/frontend/src/js/vue/components/FaqItem.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    export default class FaqItem extends Vue {
+        @Prop() intercom: string;
+        @Prop() title: string;
+        @Prop() description?: string;
+
+        public showCollapse = false;
+        public id: string = "";
+
+        mounted() {
+            this.id = Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15);
```
Comment (davyvanroy) [inline tweb/src/main/frontend/src/js/vue/components/FaqItem.vue]: what is this? why do you need it?

File: tweb/src/main/frontend/src/js/vue/components/FaqItem.vue [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        }
+
+        public get intercomArticleHtml() {
+            return Utils.getHTML(Locations.getHelpPageUrl(this.intercom));
```
Comment (davyvanroy) [inline tweb/src/main/frontend/src/js/vue/components/FaqItem.vue]: is there any reference to this?

File: tweb/src/main/frontend/src/js/vue/components/FaqItem.vue (hunk 1 above)
Comment (Simsteve7) [reply tweb/src/main/frontend/src/js/vue/components/FaqItem.vue]: The b-collapse needs a unique id, without it doesn't work. This is a quick and easy way to generate a random string.
I'll add it to Utils

File: tweb/src/main/frontend/src/js/vue/components/FaqItem.vue (hunk 2 above)
Comment (Simsteve7) [reply tweb/src/main/frontend/src/js/vue/components/FaqItem.vue]: Not yet, waiting for api or scraper.

**Pull request merged**